### DISCUSSION TITLE
Documentation pointing to dead links

### DISCUSSION THREAD

**Discussion by MrAmericanMike** [vercel/community#1452] [Idea]
This page: https://vercel.com/templates/svelte/sveltekit-authentication

contains a link to `https://sveltekit.authjs.dev/` which ends in a 404.

It's also mentioned that the Framework is Svelte, when in reality it's SvelteKit.

PS: Sorry if this is not the best place to report this, but don't know what the correct way would be if it's not here.

Thanks in advance.

**Comment by leerob** [comment]
Thanks, the link has been fixed!
